app/scripts/controllers.coffee

```diff
@@ -97,24 +97,22 @@ angular.module('memoire.controllers', ['memoire.services'])
         )
 
 
-    search = artwork.title
+    search = (artwork.title)+" "+slug(artwork.authors[0].user.first_name[0]+artwork.authors[0].user.last_name).toLowerCase()
```

Comment:
Ame api have no option "exact" search
For example when i seach "Archipel" i have 2 Aw from Giacomo Abbruzzese and one from Daphné Hérétakis named "Archipels, granites dénudés"
That why i add and author path name (ame api search on every field)